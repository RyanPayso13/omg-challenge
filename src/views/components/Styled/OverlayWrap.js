import styled from 'styled-components';

export const OverlayWrap = styled.div`
    position: absolute;
    width: 100%;
    height: 100%;
    background-color: rgba(256, 256, 256, .9);
    z-index: 10;
    display: flex;
    flex-direction: column;
`;