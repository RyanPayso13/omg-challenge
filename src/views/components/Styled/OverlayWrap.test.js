import React from 'react';
import { render } from '@testing-library/react';
import { OverlayWrap } from './OverlayWrap';

describe('<OverlayWrap />', () => {

    it('should work', () => {
        const { container } = render(<OverlayWrap />);
        expect(container.firstChild).toMatchSnapshot();    
    });

});
