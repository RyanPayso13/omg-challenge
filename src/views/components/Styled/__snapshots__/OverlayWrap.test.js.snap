// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OverlayWrap /> should work 1`] = `
<div
  class="sc-bdVaJa bqpTUz"
/>
`;
